docs/documentation.rst

```diff
@@ -0,0 +1,46 @@
+Generating the documentation locally
+----------------------------------
+
+Install the required packages
+
+.. code-block:: bash
+
+   $ pip install sphinx sphinx-autobuild sphinxcontrib-napoleon
+
+.. tip::
+   If you don't have sudo privileges, you can use the ``-- user`` flag to 
+   install the required packages locally.
+
+Enter in the docs directory and rename the existing conf.py
```

Comment:
I just double checked, and most of these instructions are actually in the current documentation already. Please have a look at https://github.com/bartvm/blocks/blob/master/docs/development/docs.rst, it should show up as a subsection "Writing and building documentation" on the "Development" page. You'll notice the use of `[docs]` (which means that we can update `setup.py` instead of the documentation) and the use of the `sphinx-build` command (which avoids having to create this `Makefile`, which is just a convenience feature of Sphinx anyway, it's not needed).

I think you could just remove this new file. Sorry!